packages/offix-client/src/links/LinksBuilder.ts

```diff
@@ -88,14 +88,15 @@ export const defaultHttpLinks = async (config: OffixClientConfig, offlineLink: A
   const localFilterLink = new LocalDirectiveFilterLink();
   links.push(localFilterLink);
 
+  const httpLinkOptions = {
+    uri: config.httpUrl,
+    ...config.httpLinkOptions
+  };
+
   if (config.fileUpload) {
-    links.push(createUploadLink({
-      uri: config.httpUrl
-    }));
+    links.push(createUploadLink(httpLinkOptions));
```

Comment:
At some point, options might diverge: https://github.com/jaydenseric/apollo-upload-client#function-createuploadlink

It is ok for now.